e2e/package.json

```diff
@@ -27,7 +27,9 @@
   },
   "jest-junit": {
     "outputDirectory": "./logs",
-    "outputName": "test-results.xml"
+    "outputName": "test-results.xml",
+    "classNameTemplate": "{filepath}",
```

Comment:
I kinda wonder why the class name and suite name use the same template.

Reply:
```
The CLI expects both filenames and classnames to be present in the timing data produced by the testing suite. By default, splitting defaults to filename, but you can specify classnames by using the --timings-type flag.
```
based on https://circleci.com/docs/2.0/parallelism-faster-jobs/#splitting-by-timing-data